subsys/logging/backends/Kconfig.net

```diff
@@ -49,6 +49,13 @@ config LOG_BACKEND_NET_AUTOSTART
 	  started by the application later on. Otherwise the logging
 	  thread might block.
 
+config LOG_BACKEND_NET_USE_DHCPV4_OPTION
+	bool "Use DHCPv4 option 7 to configure syslog server"
```

Comment:
"option 7" could be confusing w/o looking into RFC, please use option name ("Log Server Option").